process_subways.py

```diff
@@ -202,7 +205,7 @@ def calculate_centers(elements):
 
     def iterate_relation_centers_calculation(
         ignore_unlocalized_child_relations: bool,
-    ) -> List[int]:
+    ) -> list[int]:
```

Comment:
```suggestion
    ) -> list[dict]:
```
`unlocalized_relations_upd` contains `list` of `dict`